presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -72,7 +72,7 @@ This property is optional; the default is ``_schema``.
 ``mongodb.credentials``
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-A comma separated list of ``username:password@collection`` credentials.
+A comma separated list of ``username:password@authentication-database`` credentials.
```

Comment:
`authentication-database` or just `database`?
(we provide this value to `com.mongodb.MongoCredential#createCredential` which calls this just "database", but we're free to use a different wording in docs, if necessary)

is it obvious what "authentication-database" stands for, or does it require additional explanation?

Reply:
While I feel authentication-database is clear to most people, wrote additional explanation for quickstart.